### PR TITLE
DOCS-503: system.stack_trace

### PR DESCRIPTION
**Pull request by BayoNet**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation for #8344 and #8387

Description:
- Added the description of the system.stack_trace table in Russian and English.
- Restored the translated file docs/ru/sql-reference/statements/grant.md.
- Updated docs/ru/sql-reference/functions/geo/index.md.

### CODE REVIEW

File: docs/en/operations/system-tables/stack_trace.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# system.stack_trace {#system-tables_stack_trace}
+
+Contains stack traces of all server threads. Allows developers to introspect the server state.
+
+To analyze logs, use the `addressToLine`, `addressToSymbol` and `demangle` [introspection functions](../../sql-reference/functions/introspection.md).
```
Comment (blinkov) [inline docs/en/operations/system-tables/stack_trace.md:5]:
```suggestion
To analyze stack frames, use the `addressToLine`, `addressToSymbol` and `demangle` [introspection functions](../../sql-reference/functions/introspection.md).
```

File: docs/en/operations/system-tables/stack_trace.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+/build/obj-x86_64-linux-gnu/../contrib/libcxx/include/atomic:856
+/build/obj-x86_64-linux-gnu/../contrib/poco/Foundation/include/Poco/Mutex_POSIX.h:59
+/build/obj-x86_64-linux-gnu/../contrib/poco/Foundation/include/Poco/AutoPtr.h:223
+/lib/x86_64-linux-gnu/libpthread-2.27.so
+/lib/x86_64-linux-gnu/libc-2.27.so
+```
+
+**See Also**
+
+-   [Introspection Functions](../../sql-reference/functions/introspection.md) — What introspection functions are and how to use them.
```
Comment (blinkov) [inline docs/en/operations/system-tables/stack_trace.md:83]:
```suggestion
-   [Introspection Functions](../../sql-reference/functions/introspection.md) — Which introspection functions are available and how to use them.
```

File: docs/ru/sql-reference/functions/geo/index.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 toc_priority: 62
-toc_folder_title: hidden
-toc_title: Функции для работы с географическими координатами
+toc_folder_title: Гео-данные
+toc_title: hidden
```
Comment (blinkov) [inline docs/ru/sql-reference/functions/geo/index.md:4]: unrelated change

File: docs/ru/sql-reference/statements/grant.md [hunk 4]
```diff
@@ -1,474 +1,479 @@
----
-toc_priority: 39
-toc_title: GRANT
----
+# GRANT
```
Comment (blinkov) [inline docs/ru/sql-reference/statements/grant.md:1]: unrelated change

**Review by blinkov — CHANGES_REQUESTED** [review]

File: docs/ru/sql-reference/functions/geo/index.md (hunk 3 above)
Comment (BayoNet) [reply docs/ru/sql-reference/functions/geo/index.md:4]: Ok, I'll try to split PR's more carefully in the future.
